tools/biotransformer/biotransformer.xml

```diff
@@ -51,43 +38,46 @@
     <param name="tolerance" type="float" value="0.01" label="Mass tolerance for metabolite identification."/>
 
     <conditional name="input">
-    	<param name="type_selector" type="select" label="Input format [SMILES, SDF, CSV].">
-        	<option value="smile" selected="true">SMILES</option>
-		<option value="sdf">SDF</option>
-		<option value="csv">CSV</option>
+    	<param name="type_selector" type="select" label="Input format [CSV].">
+		<option value="csv" selected="true">CSV</option>
         </param>
-        <when value="sdf">
-        	<param name="input_format" argument="-isdf" type="data" format="sdf" label="SDF"/>
-    	</when>
-    	<when value="smile">
-        	<param name="input_format" argument="-ismi" type="text" label="SMILES (example string format: CC(C)C1=CC=C(C)C=C1O )."/>
-    	</when>
 	<when value="csv">
-		<param name="input_format" argument="-csv" type="data" format="csv" label="CSV with one molecule (SMILES format) per line."/>
+		<param name="input_format" argument="-icsv" type="data" format="csv" label="CSV with one molecule (SMILES format) per line."/>
 	</when>
     </conditional>
 
+    <!-- If we would like to allow user to choose between format - not available now
```

Comment:
Same here, please drop the commented out code, you can go back for it any time in the future using git.